README.md

```diff
@@ -13,7 +13,11 @@
 
 ## What is ``PAINT``
 
-PAINT is a FAIR database for Concentrating Solar Power plants (CSP).
+The PAINT database makes operational data of concentrating solar power plants available in accordance with the FAIR data
+principles, i.e., making them findable, accessible, interoperable, and reusable. Currently, the data encompasses
+calibration images, deflectometry measurements, kinematic settings, and weather information of the concentrating solar
+power plant in Jülich, Germany, with the global power plant id (GPPD) WRI1030197. Metadata for all database entries
+follow the spatio-temporal asset catalog (STAC) standard.
```

Comment:
Do we want singular or plural "data"?

Reply:
I always get confused with this - I like it the way it is, although I would change the "follow" to "follows" in the last sentence.